packages/serializer/tests/serializer.test.ts

```diff
@@ -63,4 +63,28 @@ describe('Serializer multiple resource', () => {
     expect(meta.totalCount).toEqual(99);
     expect(meta.planet).toEqual('earth');
   });
+
+  test('Should skip serialization entirely if skipSerialization is true', async () => {
+    const rawData = [
+      { firstName: 'John', lastName: 'Doe' },
+      { firstName: 'Jane', lastName: 'Doe' },
+    ];
+
+    const userSerializer = new Serializer('user', {
+      attributes: [],
+    }, { skipSerialization: true });
```

Comment:
I'd love if it could be used like this

```typescript
    const userSerializer = new Serializer('user', { skip: true });
```

I don't think we should require `attributes` if we're skipping the serialization. It's also more clear that we're doing a no-op. But this seems to be a bigger change with how `ISerializerConfig` is provided, so maybe not worth it for now

Reply:
I agree, but just wanted to keep the changes simple to comply with your request 😂 Will check it out though